wandb/sdk/data_types/html.py

```diff
@@ -74,35 +74,35 @@ def inject_head(self) -> None:
         self.html = join.join(parts).strip()
 
     @classmethod
-    def get_media_subdir(cls: Type["Html"]) -> str:
+    def _get_media_subdir(cls: Type["Html"]) -> str:
         return os.path.join("media", "html")
 
-    def to_json(self, run_or_artifact: Union["LocalRun", "Artifact"]) -> dict:
-        json_dict = super().to_json(run_or_artifact)
+    def _to_json(self, run_or_artifact: Union["LocalRun", "Artifact"]) -> dict:
```

Comment:
is there a way to hide these functions without changing the name? it feels too risky for a pr that does a doc change

Reply:
Yeah, @timoffex made the same point. I left a comment on his comment for a possible workaround. Let me know what ya'll think.